sip-quickstart/assets/admin/admin-client.js

```diff
@@ -32,14 +29,16 @@ class AdminClient {
     return response.json();
   }
 
+  async checkAdminPassword() {
+    const response = await fetch('./check-adminPassword');
```

Comment:
Can we keep this in dasherized style please? `check-admin-password`

Reply:
Done, thanks for pointing that out!